### PR TITLE
Add Visual Basic template

### LINKED ISSUE #689: Add Visual Basic dotnet new project template
Should match VB project template from Framework

* WinFormsApplication-VBasic
  * Company.WinFormsApplication1.vbproj
  * Form1.Desiger.vb
  * Form1.vb
  * App.config (?)
  * My Project directory
    * Application.Designer.vb
    * Application.myapp (?)
    * AssemblyInfo.vb (? pretty sure no)
    * Resources.Designer.vb (?)
    * Resources.resx (?)
    * Settings.Designer.vb (?)
    * Settings.settings (?)
  * (? missing) xlf directory

And we'll need to figure out the template.config for this

### PR DESCRIPTION
**Pull request by Berrysoft**
fixes #689
The `Form1.resx` file and `xlf` folder are copied from C# template.

### CODE REVIEW

**Comment by Berrysoft** [comment]
image
The preview3 template package doesn't contain the `xlf` folder, is it by design?

File: pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/Microsoft.Dotnet.Winforms.ProjectTemplates.csproj [hunk 1]
```diff
@@ -41,8 +41,7 @@
   <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
     <Content Include="content\**\*" />
     <Content Update="@(Content)">
-      <!-- Supress creation of content directory -->
-      <PackagePath>.\</PackagePath>
+      <PackagePath>.\content\</PackagePath>
```
Comment (zsd4yr) [inline pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/Microsoft.Dotnet.Winforms.ProjectTemplates.csproj]: why was this changed? Why remove the comment?

**Comment by zsd4yr** [comment]
> The preview3 template package doesn't contain the `xlf` folder, is it by design?

@Berrysoft I believe the `xlf` folder was added when the project templates were moved to this repo; before this, including the xlf was impossible

File: pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/Microsoft.Dotnet.Winforms.ProjectTemplates.csproj (hunk 1 above)
Comment (Berrysoft) [reply pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/Microsoft.Dotnet.Winforms.ProjectTemplates.csproj]: See [here](https://github.com/dotnet/templating/blob/stabilize/template_feed/Microsoft.DotNet.Common.ProjectTemplates.3.0/Microsoft.DotNet.Common.ProjectTemplates.3.0.csproj), only to be same as other templates.

**Comment by Berrysoft** [comment]
image
I packed it by myself and it contains `xlf` folder. Also, after running `dotnet new winforms`, the `xlf` folder will be copied to the project created. Is there something wrong?

**Comment by zsd4yr** [comment]
> Is there something wrong?

no that looks good

File: pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/Microsoft.Dotnet.Winforms.ProjectTemplates.csproj (hunk 1 above)
Comment (zsd4yr) [reply pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/Microsoft.Dotnet.Winforms.ProjectTemplates.csproj]: This is not right. I am reverting this change. 

These results should not be nested in a content folder

Comment (zsd4yr) [reply pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/Microsoft.Dotnet.Winforms.ProjectTemplates.csproj]: #750

Comment (Berrysoft) [reply pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/Microsoft.Dotnet.Winforms.ProjectTemplates.csproj]: OK, thanks!
